controller/model/api_session_certificate_manager.go

```diff
@@ -89,8 +89,6 @@ func (self *ApiSessionCertificateManager) CreateFromCSR(apiSessionId string, lif
 		PEM:          string(certPem),
 	}
 
```

Comment:
I don't know why this was here, but it has no effect, so removing it should be ok